File: leetcode/1easy/이경호/1374.cpp
class Solution {
public:
  string generateTheString(int n) {
    string front = "a";
    string back = "b";
    string result;
    if (n % 2 == 1) {
      for (int i = 0; i < n; i++) {
        result += front;
      }
    }
    else {
      for (int i = 0; i < n - 1; i++) {
        result += front;
      }
      result += back;
    }
    return result;
  }
};

File: leetcode/1easy/박은비/1374.swift
<<<<<<< HEAD
class Leetcode.이재훈.이재훈.Solution {
=======
class Leetcode.이재훈.Solution {
>>>>>>> ab7f46f73d8f18bcd38c5cf06ecabae64f53a424
    func generateTheString(_ n: Int) -> String {
        
        let count = n % 2 == 0
        ? 1
        : 0

        var answer = String(repeating: "a", count: count)
        answer += String(repeating: "z", count: n-count)

        return answer

    }
}
